provider/terraform/tests/resource_compute_project_metadata_test.go

```diff
@@ -34,6 +34,11 @@ func TestAccComputeProjectMetadata_basic(t *testing.T) {
 					testAccCheckComputeProjectMetadataSize(projectID, 2),
 				),
 			},
+			resource.TestStep{
```

Comment:
Since you have these now, you can get rid of (I think) all of the Checks in the TestSteps

Reply:
Done.